dpgen/generator/run.py

```diff
@@ -3733,6 +3733,20 @@ def run_iter (param_file, machine_file) :
                 raise RuntimeError ("unknown task %d, something wrong" % jj)
             record_iter (record, ii, jj)
 
+        
+def get_atomic_masses(chemical_symbol):
+  chemical_symbols = ['H', 'He', 'Li', 'Be', 'B', 'C', 'N', 'O', 'F', 'Ne', 'Na', 'Mg', 'Al', 'Si', 'P', 'S', 'Cl', 'Ar', 'K', 'Ca', 'Sc', 'Ti', 'V', 'Cr', 'Mn', 'Fe', 'Co', 'Ni', 'Cu', 'Zn', 'Ga', 'Ge', 'As', 'Se', 'Br', 'Kr', 'Rb', 'Sr', 'Y', 'Zr', 'Nb', 'Mo', 'Tc', 'Ru', 'Rh', 'Pd', 'Ag', 'Cd', 'In', 'Sn', 'Sb', 'Te', 'I', 'Xe', 'Cs', 'Ba', 'La', 'Ce', 'Pr', 'Nd', 'Pm', 'Sm', 'Eu', 'Gd', 'Tb', 'Dy', 'Ho', 'Er', 'Tm', 'Yb', 'Lu', 'Hf', 'Ta', 'W', 'Re', 'Os', 'Ir', 'Pt', 'Au', 'Hg', 'Tl', 'Pb', 'Bi', 'Po', 'At', 'Rn', 'Fr', 'Ra', 'Ac', 'Th', 'Pa', 'U', 'Np', 'Pu', 'Am', 'Cm', 'Bk', 'Cf', 'Es', 'Fm', 'Md', 'No', 'Lr', 'Rf', 'Db', 'Sg', 'Bh', 'Hs', 'Mt', 'Ds', 'Rg', 'Cn', 'Nh', 'Fl', 'Mc', 'Lv', 'Ts', 'Og']
+
+  # doi:10.1515/pac-2015-0305
```

Comment:
Why not use the latest data?
https://doi.org/10.1515/pac-2019-0603

Reply:
updated.

Reply:
I can not fully understand [wanghan-iapcm](https://github.com/wanghan-iapcm)'s comment below.  I have rewritten the `get_atomic_masses ` function, and now it can be called by three types, _i. e._ `element_names`, `chemical_symbols` and `atomic_number`. I am not sure if this will remove his/her concern.

Reply:
I mean atom name can be ANY string, not necessary element_names or chemical_symbols. 
For example it is valid to write `"type_map" : ["foo", "bar"]`.

Reply:
I agree with @wanghan-iapcm. This should be documented that `auto` only supports the standard atoms. (For example, `OW` will not be supported)

Reply:
I updated the documentation.